di-ipv-credential-issuer-stub/src/main/resources/templates/authorize.mustache

```diff
@@ -253,6 +253,13 @@
                     </h1>
                 </legend>
 
+                <div class="govuk-form-group">
+                    <input type="checkbox" name="vcExpiryFlg" id="vcExpiryFlg" checked>
+                    <label class="govuk-label">
+                        Expiry
```

Comment:
Maybe "Include expiry on VC" to be a little clearer?

Also not a big deal but you could make this a gov.uk checkbox - https://design-system.service.gov.uk/components/checkboxes/

Reply:
Change name to 'Include VC expiry'
Switched to gov.uk checkbox.